tools/get_platform.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
```

Comment:
Shouldn't this change to 
```suggestion
#!/usr/bin/env python3
```
?